lib/recipe.rb

```diff
@@ -27,6 +27,12 @@ def ingredient_entries
     end
   end
 
+  def ingredients
+    recipe_data["ingredient_entries"].map do |ingredient_entry|
+      ingredient_entry["ingredient"]
+    end
```

Comment:
Could be interesting to return an `Ingredient` object here. We could do things like group ingredients together:

```ruby
class RecipeBook # or whatever name
  def ingredients
    [
      Ingredient.new("lime"),
      SynonymGroup.new([Ingredient.new("bacon"), Ingredient.new("pancetta")]),
      Ingredient.new("chicken stock"),
    ]
  end
end
```
I imagine then given an input ingredient we'd iterate through this list until we find the first one that matches, then use that as our canonical ingredient object. We could do the same when creating a `Recipe` object.
e.g.

```ruby
class Ingredient
  def initialize(name)
    @name = name
  end
  def match?(name)
    @name =~ name # can do case insensitive matches here too
  end
end

class SynonymGroup
  def initialize(ingredients)
    @ingredients = ingredients
  end
  def match?(name)
    @ingredients.any? { |ingredient| ingredient.match?(name) }
  end
end
```